### PR TITLE
fix: Added radar chart displaying capabilities on task types

### PR DESCRIPTION
**Pull request by x-tabdeveloping**
I added a radar chart to a tab you can switch to from the other plot.
It only displays the top 5 models that have been run on all task types to avoid clutter.
I also decided not to display instruction retrieval, since it can be negative and it looks weird.
Here's how it looks:
image

### CODE REVIEW

**Comment by x-tabdeveloping** [comment]
#1443

File: mteb/leaderboard/figures.py [hunk 1]
```diff
@@ -97,3 +97,90 @@ def performance_size_plot(df: pd.DataFrame) -> go.Figure:
...
+    "BitextMining",
+    "Classification",
+    "MultilabelClassification",
+    "Clustering",
+    "PairClassification",
+    "Reranking",
+    "Retrieval",
+    "STS",
+    "Summarization",
+    # "InstructionRetrieval",
```
Comment (isaac-chung) [inline mteb/leaderboard/figures.py:113]: Could include the reason you shared here.

File: mteb/leaderboard/figures.py [hunk 2]
```diff
@@ -97,3 +97,90 @@ def performance_size_plot(df: pd.DataFrame) -> go.Figure:
         margin=dict(b=20, t=10, l=20, r=10),  # noqa
     )
     return fig
+
+
+TOP_N = 5
```
Comment (isaac-chung) [inline mteb/leaderboard/figures.py:102]: May be moved lower and make it depend on something we'll change if we were to increase top N?
```suggestion
TOP_N = len(line_colors)
```

**Review by isaac-chung — APPROVED** [review]
Looks great! Added 2 suggestions.

File: mteb/leaderboard/figures.py (hunk 1 above)
Comment (x-tabdeveloping) [reply mteb/leaderboard/figures.py:113]: Sure thing

**Pull request merged**